Rabbitmq to bionic

ryanmriddell opened the pull request:
Swapped out Xenial for Bionic. Updated dependencies to match latest useable versions:

Cf-rabbitmq-multitenant-broker moved from 103.0.0 =>  107.0.0
Cf-cli moved from 1.33 => 1.36
Cf-rabbitmq-smoke-tests-genesis-releases moved from 106.0.0 => 120.0.0
Cf-rabbitmq moved from  410.0.0 => 416.0.0
Bpm moved from 1.1.15 => 1.1.16
Haproxy moved from 11.9.2 => 11.9.3
Loggregator-agent stays with 6.3.8 => 6.3.8
Rabbitmq metric emitter release 0.4.1 => 0.4.1
Routing-release 0.228.0 => 0.228.0
[Jira Ticket Link](https://starkandwayne.atlassian.net/browse/OCF-176)

Commit:
RR Initial Commit - Changed out xenial wording to bionic

Commit:
RR - modified more spec tests, 7 of 9 now working

Commit:
RR - All tests currently pass

Commit:
RR - updated dependency versions to latest

Commit:
RR- commented out but didnt delete sha1s

Commit:
RR - added the right SHA1s for all newest dependencies

Commit:
RR - changed cf-rabbitmq to 416 because of strange difficulty with 418

Commit:
repo demanding I put the DS Store modifications in there too

Commit:
RR - changed out dependencies in results folder to match new updated dependencies added previously

ryanmriddell committed:
Merge branch 'master' into rabbitmq-to-bionic

norman-abramovitz commented on manifests/releases/cf-cli.yml:
```diff
@@ -2,6 +2,6 @@
-    version: 1.33.0
-    url: https://bosh.io/d/github.com/bosh-packages/cf-cli-release?v=1.33.0
-    sha1: c038fdb8e9c0707dc407bf0f2846ad324a5d8d9b
+    version: 1.36.0
```
One of the things this package does is add the cf v6, cf v7, and cf v8 to the release.   How is s the cf command being used in this bosh release?    Are there any tests that might need to be updated using the different versions of the CF command?

Commit:
RR - Changes made during pair time with Dr Gao

Commit:
"merged branches"
Merge branch 'rabbitmq-to-bionic' of https://github.com/genesis-community/rabbitmq-genesis-kit into rabbitmq-to-bionic

Commit:
RR added changes to route-registrar yml files and I believe I changed some things back as well
